.github/workflows/test.yml

```diff
@@ -34,11 +34,12 @@ jobs:
           toolchain: stable
           profile: minimal
           default: true
+      - name: Install Hatch
+        run: |
+          pip install hatch
       - name: Build Ypy
         run: |
-          pip install maturin
-          maturin develop
+          hatch run maturin develop
       - name: Run Tests
         run: |
-          pip install pytest
-          pytest
+          hatch run pytest
```

Comment:
I'm not sure what happens if we try `hatch run test:pytest` here, presumably since there's only one version of Python on the runner then it's effectively the same as `hatch run pytest`.